docs/architecture/adr-010-unified-cli-arguments-hermes.md

```diff
@@ -0,0 +1,173 @@
+
+__Commands for keys__
+
+* `keys add --chain <CHAIN_ID> --key-file <KEY_FILE> --mnemonic-file <MNEMONIC_FILE>`
+    * Optional: `[--hd-path <HD_PATH>] [--key-name <KEY_NAME>]`
+
+* `keys balance --chain <CHAIN_ID>`
+    * Optional: `[--key-name <KEY_NAME>]`
+
+* `keys delete --chain <CHAIN_ID> --all`
+
+* `keys delete --chain <CHAIN_ID> --key-name <KEY_NAME>`
+
+* `keys list --chain <CHAIN_ID>`
+
+__Listen__
+
+* `listen --chain <CHAIN_ID>`
+    * Optional: `[--event <EVENT>]`
+
+__Misbehaviour__
+
+* `misbehaviour --chain <CHAIN_ID> --client <CLIENT_ID>`
+
+__Start the relayer in multi-chain mode__
+
+* `start`
+    * Optional: `[--full-scan]`
+
+__Clear objects__
+
+* `clear packets --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID>`
+
+__Queries__
+
+* `query channel client --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID>`
+
+* `query channel end --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID>`
+    * Optional: `[--height <HEIGHT>]`
+
+* `query channel ends --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID>`
+    * Optional: `[--height <HEIGHT>] [--verbose]`
+
+* `query channels --chain <CHAIN_ID>`
+    * Optional: `[--dst-chain <DST_CHAIN_ID>] [--verbose]`
+
+* `query client connections --chain <CHAIN_ID> --client <CLIENT_ID>`
+    * Optional: `[--height <HEIGHT>]`
+
+* `query client consensus --chain <CHAIN_ID> --client <CLIENT_ID>`
+    * Optional: `[--consensus-height <CONSENSUS_HEIGHT>] [--height <HEIGHT>] [--heights-only]`
+
+* `query client header --chain <CHAIN_ID> --client <CLIENT_ID> --consensus-height <CONSENSUS_HEIGHT>`
+    * Optional: `[--height <HEIGHT>]`
+
+* `query client state --chain <CHAIN_ID> --client <CLIENT_ID>`
+    * Optional: `[--height <HEIGHT>]`
+
+* `query clients --chain <CHAIN_ID>`
+    * Optional: `[--omit-chain-ids] [--src-chain <ID>]`
+
+* `query connection channels --chain <CHAIN_ID> --conn <CONNECTION_ID>`
+
+* `query connection end --chain <CHAIN_ID> --conn <CONNECTION_ID>`
+    * Optional: `[--height <HEIGHT>]`
+
+* `query connections --chain <CHAIN_ID>`
+
+* `query packet ack --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID> --seq <SEQUENCE>`
+    * Optional: `[--height <HEIGHT>]`
+
+* `query packet acks --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID>`
+
+* `query packet commitment --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID> --seq <SEQUENCE>`
+    * Optional: `[--height <HEIGHT>]`
+
+* `query packet commitments --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID>`
+
+* `query packet pending --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID>`
+
+* `query packet unreceived-acks --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID>`
+
+* `query packet unreceived-packets --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID>`
+
+* `query tx events --chain <CHAIN_ID> --hash <HASH>`
+
+__Shell completion__
+
+* `completions --shell <SHELL>`
+
+__Validate configuration file__
+
+* `config validate`
+
+__Health check__
+
+* `health-check`
+
```

Comment:
You seemed to have forgotten to include `hermes tx raw *` commands, right?

On that note, can we remove the `raw`? There are no other possibilities other than `raw`, and to me `hermes tx <tx>` is very clear what it does. We can even document `hermes tx` as being able to send "raw ibc datagrams".

Reply:
I second this. `raw` needs to be dropped.

Reply:
> You seemed to have forgotten to include `hermes tx raw *` commands, right?

iirc, we discussed to leave the `raw` CLIs for later. I think we should do that in a subsequent PR (we can add them to the same ADR). This because there are a lot of  `raw` commands and there is a lot more work that needs be done there. They are also not used as much by operators.

Reply:
I have the `raw` CLIs ready, as soon as this PR is merged I can create the one with the `raw` commands.

Reply:
@ljoss17 can you open an issue so that we can track it?